Sources/PaymentButtons/ImageAsset.swift

```diff
@@ -18,16 +18,10 @@ enum ImageAsset {
             imageAssetString += "credit_"
         }
 
-        switch button.color {
-        case .gold, .white, .silver:
-            imageAssetString += "color"
-
-        case .black, .darkBlue:
-            imageAssetString += "monochrome"
-
-        case .blue:
-            imageAssetString += "blue"
-        }
+	switch button.color {
+	case .gold, .white, .silver, .black, .blue, .darkBlue:
+	    imageAssetString += "color"
+	}
```

Comment:
qq: Do we need this switch? Looking over PaymentButtonColor, it seems that we are covering all the possibilities, which makes this appear unnecessary
```suggestion
	imageAssetString += "color"
```

Reply:
Good point!